pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -239,8 +240,11 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 	reqLogger.Infof("Evict all Pods from Node: %s", nodeName)
 
 	if err = drain.RunNodeDrain(r.drainer, nodeName); err != nil {
-		return r.reconcileAndError(instance, err)
+		reqLogger.Infof("not all pods evicted: %s : %v", nodeName, err)
+		waitOnReconcile := time.Duration(WaitOnDrainErrorInSeconds * time.Second)
```

Comment:
no need to cast to duration, it's implicit by multiplying with time.Second
and if we use a `WaitOnDrainError` duration, it's even easier

Reply:
done